RecoEgamma/PhotonIdentification/data/PHYS14/effAreaPhotons_cone03_pfChargedHadrons.txt

```diff
@@ -0,0 +1,13 @@
+# This file contains Effective Area constants for 
```

Comment:
similar to mvaElectronID_PHYS14_PU20bx25_nonTrig_V1_cff.py, is this file needed or can it be used in this release?

Reply:
I don't know, since I don't maintain the effective areas. But it was in the externals before anyway, so it's not like it's added back (as is the case for the 2014 electron MVA).

Reply:
I think it's interesting to keep for comparisons. It's the only Run2 MVA ID which doesn't include conversion rejection yet. But if you think there is no place for it here I'll just keep it privately.